<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="vprybeha-w7" ignored="0" tests="2" failures="1" timestamp="28 Dec 2018 13:51:38 GMT" time="57.353" errors="0">
  <testcase name="firsttest" time="39.395" classname="TestClass"/>
  <testcase name="secondtest" time="3.205" classname="TestClass">
    <failure type="SupportClasses.NewException" message="Assertion finished false">
      <![CDATA[SupportClasses.NewException: Assertion finished false
at TestClass.secondtest(TestClass.java:36)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- secondtest -->
</testsuite>
